### PR TITLE
Feature/275691446/update project requirements

### PR DESCRIPTION
**Pull request by soulless-viewer**
### Description

This PR was opened to clarify our project requirements.
Not all of them are accurate and justified, which can cause bugs or at least misunderstandings.

### Changes

1. Minimum required Python version was downgraded from **3.9** _(unjustified)_ to **3.7**
   * _Python versions **<3.7** are [not officially supported by dependent libraries](https://github.com/googleapis/google-api-python-client#unsupported-python-versions). Their use can lead to unpredictable errors._
2. The Python version used in GitHub flow was also set as **3.7** to test on the oldest version
3. Minimum required `google-api-python-client` version was set as **2.68.0** *
   * _Technically, the version should be at least **2.1.0**, since it introduced resources on which we depend. However, it was decided to use the version that came out on a par with the beginning of our development_
4. Minimum required `google-auth` version was set as **2.15.0** *
   * _Also, it was decided to use the version that came out on a par with the beginning of our development_

### CODE REVIEW

File: README.md
Comment (halleysouza) [inline README.md:1]: Could you pls update the last shellscript block, where the format (summary, progressbar, etc) is not the current one.

**Review by halleysouza — CHANGES_REQUESTED** [review]
Thanks, please have a look on the README.md comment.

File: README.md
Comment (soulless-viewer) [reply README.md:1]: @halleysouza 
Let's fix Readme in another one PR to keep this one only for requirements

Comment (halleysouza) [reply README.md:1]: sgtm!

**Review by halleysouza — APPROVED** [review]
lgtm

**Pull request merged**